system/Database/Postgre/Connection.php

```diff
@@ -128,10 +128,12 @@ public function getVersion(): string
             return $this->dataCache['version'];
         }
 
-        if (! $this->connID || ($pgVersion = pg_version($this->connID)) === false) {
+        if (! $this->connID) {
             $this->initialize();
         }
 
+        $pgVersion = pg_version($this->connID);
+
         return isset($pgVersion['server']) ? $this->dataCache['version'] = $pgVersion['server'] : false;
```

Comment:
The interface is `public function getVersion(): string;`. So `false` should not be returned.